static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -715,7 +709,7 @@ function WidgetBuilder({
                         inline={false}
                         flexibleControlStateSize
                         stacked
-                        error={state.errors?.[queryIndex]?.conditions}
+                        error={state.errors?.queries?.[queryIndex]?.conditions}
```

Comment:
![Screen Shot 2022-02-23 at 4 02 17 PM](https://user-images.githubusercontent.com/22846452/155430295-7473001a-f470-4351-8e13-8bff7499196f.png)

![Screen Shot 2022-02-23 at 4 02 10 PM](https://user-images.githubusercontent.com/22846452/155430290-453b74e8-0a2a-4c7a-9858-19a84353a020.png)

Reply:
We show the errors in tooltips when the cursor is inside of these aswell right?

Reply:
not really... these tooltips are shown right when we click on the submit button

Reply:
![example-evan](https://user-images.githubusercontent.com/29228205/155506611-883df05b-858f-4e61-ac6b-82accd605bb9.gif)

Reply:
I mean on the search token elements in the screenshot :)

Reply:
![Screen Shot 2022-02-24 at 9 44 50 AM](https://user-images.githubusercontent.com/22846452/155546401-fbc6b845-9e1c-4ac3-abc1-588c6af23e6f.png)

Ah, yeah, we also show the message here when a user's text cursor is in the search token